br/pkg/restore/split.go

```diff
@@ -111,7 +111,7 @@ SplitRegions:
 			}
 			return errors.Trace(errScan)
 		}
-		splitKeyMap := getSplitKeys(rewriteRules, sortedRanges, regions)
+		splitKeyMap := getSplitKeys(rewriteRules, sortedRanges, regions, true)
```

Comment:
this is used for txn split too.
so this cannot simply be true.

Reply:
I'm sorry for the mistake...
Fixed, PTAL, thanks~